jetstream/workflows/run.yaml

```diff
@@ -10,25 +10,62 @@ spec:
     - name: project_id
     - name: dataset_id
     - name: bucket
+    - name: recreate_enrollments
   templates:
   - name: jetstream
     parallelism: 5  # run up to 5 containers in parallel at the same time
     inputs:
       parameters:
         - name: experiments
     steps:
-    - - name: analyse-and-export
-        template: analyse-and-export
+    - - name: ensure-enrollments-and-analyze
+        template: ensure-enrollments-and-analyze
         arguments:
           parameters:
           - name: slug
             value: "{{item.slug}}"
-          - name: date
-            value: "{{item.date}}"
+          - name: dates
+            value: "{{item.dates}}"
         withParam: "{{inputs.parameters.experiments}}"  # process these experiments in parallel
         continueOn:
           failed: true
 
+  - name: ensure-enrollments-and-analyze
+    inputs:
+      parameters:
+        - name: slug
+        - name: dates
+    steps:
+      - - name: ensure-enrollments
+          template: ensure-enrollments
+          arguments:
+            parameters:
+              - name: slug
+                value: "{{inputs.parameters.slug}}"
+      - - name: analyse-and-export
+          template: analyse-and-export
+          arguments:
+            parameters:
+              - name: slug
+                value: "{{inputs.parameters.slug}}"
+              - name: date
+                value: "{{item}}"
+          withParam: "{{inputs.parameters.dates}}"
+
+  - name: ensure-enrollments
+    inputs: 
+      parameters:
+      - name: slug  
+    container:
+      image: gcr.io/moz-fx-data-experiments/jetstream:latest
+      command: [
+        jetstream, --log_to_bigquery, ensure-enrollments, 
+        "--experiment_slug={{inputs.parameters.slug}}", 
+        "--dataset_id={{workflow.parameters.dataset_id}}", 
+        "--project_id={{workflow.parameters.project_id}}",
+        "--recreate-enrollments={{workflow.parameters.recreate_enrollments}}"
+      ]
+
```

Comment:
I thought it might be easier to understand these changes by visualizing them as what the graph would look like in Argo.

For daily runs the graph would look like:
![cache-enrollments](https://user-images.githubusercontent.com/1239705/109051636-8e9d3380-768f-11eb-8c00-285d6b7df353.png)

The ensure-enrollment step would just verify that the enrollments table exist, unless it is the first time the experiment is analyzed then the step will create the enrollments table.

When rerunning an experiment the graph will look like:
![cache-enrollments-rerun](https://user-images.githubusercontent.com/1239705/109051670-96f56e80-768f-11eb-9294-432b4bfb918b.png)

Once the enrollments table has been created or its existence has been verified, analyses for the days in the active time period will be rerun.